### ISSUE TITLE
`aria-controls` attribute from element with role "scrollbar" (ARIA 1.2)

### ISSUE THREAD

**Issue by Zidious** [dequelabs/axe-core#3146]
Ref: https://github.com/dequelabs/axe-core-npm/issues/318

`aria-controls` attribute on `form` element [ARIA 1.2 - Scrollbar](https://www.w3.org/TR/wai-aria-1.2/#scrollbar)

[W3C validation file used](https://github.com/w3c/aria/blob/main/validator-tests/scrollbar-role-aria-controls.html)

Note: it is currently a `allowedAttrs` not a `requiredAttrs` within axe-core due to lack of screen reader support across multiple browsers.
